source/blog/2018-02-20-Your-Container-Volumes-Served-By-oVirt.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Your Container Volumes Served By oVirt 
+author: rgolan
+tags: oVirt, oVirt 4.2, open source virtualization,kubernetes, openshift, external-storage, flexdriver, provisioner
+date: Tue Feb 20 08:54:34 IST 2018
+---
+
+> Note: _< 5 minutes read_
+
+When running a virtualization workload on oVirt, a VM disk is 'natively' a disk somewhere on your network-storage.  
+Entering containers world, on k8s or Openshift, there are many options specifically because the workload can be totally stateless, i.e
+they are stored on a host supplied disk and can be removed when the container is terminated. The more interesting case are *stateful workloads* i.e apps that persist data (think DBs, web servers/services, etc). k8s/openshift designed an api to dynamically provision the container storage (volume in k8s terminology). 
+
+See the [resources](#resources) section for more details. 
+
+In this post I want to cover how oVirt can provide volumes for containers running on k8s/openshift cluster.
+
+READMORE
+
+## Overview
+Consider this: you want to deploy wikimedia as a container, with all its content served from `/opt`. 
+For that you will create a persistent volume for the container - when we have state to keep and server
+creating a volume makes sense. It is persistent, it exists regardless the container state,
+and you can choose which directory exactly you serve that volume, and that is the most important
+part, k8s/openshift gives you an Api to determine who will provide the volume that you need.
```

Comment:
Api -> API.
Also, in other places you use 'api'.

Reply:
done